### ISSUE TITLE
add an openapi spec to design/document the api

### ISSUE THREAD

**Issue by sonalranjit** [CivicTechTO/toronto-bids#6]
It would be great to have an open api doc to collaboratively design and document on an open api specification document which can be implemented when the server is ready.

**Comment by Arpanio** [comment]
I haven't worked on this before. Miranda seems to be quite a few commits ahead in her local repo. Maybe we should pull her in here.

**Comment by sonalranjit** [comment]
Yes she should commit her code into this repo, https://swagger.io/docs/specification/basic-structure/
We can design the api with an open-api spec before we ever start coding. The spec can be written in yaml file so it can be committed here into this repo.

**Comment by alexwolson** [comment]
Closing as obsolete: the server this spec was for doesn't exist.

This asked for an OpenAPI spec to design/document the API "which can be implemented when the server is ready." The Clojure API server (`app/server`) was removed in #60 — it had been dormant since mid-2023 and never reached that point.

The goal behind it is still alive, but it's answered differently now: `tb export` writes the whole store to a single solicitation-centric JSON artifact, which is the current mechanism for programmatic access. If an HTTP API gets built on top of that later, an OpenAPI spec would be worth a fresh issue written against that design.
